### PR TITLE
ui changes on home page

### PR DESCRIPTION
**Pull request by olek557**

### CODE REVIEW

File: app/assets/javascripts/templates/profile.html [hunk 1]
```diff
@@ -95,27 +95,27 @@
...
           </div>
         </div>
       </div>
     </div>
   </div>
 </div>
-</div>
 
-<md-tabs md-dynamic-height=""
+<!-- <md-tabs md-dynamic-height=""
```
Comment (okondriv) [inline app/assets/javascripts/templates/profile.html]: Do we need this commented code?

File: app/assets/stylesheets/partials/layout.scss [hunk 2]
```diff
@@ -40,10 +40,10 @@ a:hover {
 }
 
 .bc-section {
-  margin-top: 70px;
-  & + .bc-section {
-    margin-top: 20px;
-  }
+  margin-top: 20px;
+  // & + .bc-section {
```
Comment (okondriv) [inline app/assets/stylesheets/partials/layout.scss]: Please remove all commented code.

**Comment by olek557** [comment]
fixed

File: app/assets/stylesheets/partials/card.scss [hunk 3]
```diff
@@ -1,5 +1,5 @@
 .bc-card {
-  border-radius: 10px;
+  // border-radius: 10px;
```
Comment (okondriv) [inline app/assets/stylesheets/partials/card.scss]: Please remove this commented code as well.

File: app/assets/stylesheets/partials/card.scss [hunk 4]
```diff
@@ -21,13 +21,17 @@
     margin-bottom: 30px;
     width: $card-width;
     background: none;
-    box-shadow: none;
+    // box-shadow: none;
```
Comment (okondriv) [inline app/assets/stylesheets/partials/card.scss]: And this :).

**Pull request merged**